rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -456,8 +456,7 @@ def shutdown(self) -> None:
 
 def main():
     play_selector = basic_play_selector.BasicPlaySelector()
-
-    # change this line to test different plays (set to None if no desired test play)
+    # play_selector = None
```

Comment:
should put back the comment that was here originally (can simply `git checkout` the file from the base branch)